tests/runtime_test.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import pytest
+import sys
+import os
+
+
+def main():
+    if sys.version_info < (3, 0):
+        version_dirs = [
+            '2and3',
+            '2',
+        ]
+    else:
+        version_dirs = [
+            '2and3',
+            '3',
+            '%d.%d' % (sys.version_info[0], sys.version_info[1]),
```

Comment:
This may be true, but it's certainly confusing, since you are using the same directory naming scheme as for the stubs, and for stubs the rule that Jelle explains is essential (otherwise e.g. asyncio would not be found for Python 3.5 or higher).

It also points, again, to yet another reason why I don't believe these runtime tests are going to help much (it's not checking that the `sys.version_info` checks in the stubs are correct).